### ISSUE TITLE
Syntax highlighting theme doesn't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by rawrmonstar** [FormidableLabs/component-playground-docs#12]
Our syntax highlighting such as: 
![image](https://cloud.githubusercontent.com/assets/13814048/18973671/93e4def0-8653-11e6-8053-f2333a47c698.png)
doesn't meet minimum accessibility contrast ratios according to aXe.
